model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -27,9 +27,11 @@ import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.Project
 import org.ossreviewtoolkit.model.Provenance
 import org.ossreviewtoolkit.model.RepositoryProvenance
+import org.ossreviewtoolkit.model.SourceCodeOrigin
```

Comment:
Commit message:
- In "only contains  the" I believe "only" can be dropped, as it's mentioned earlier in the sentence already, and the duplicate space should be removed.